src/base/bittorrent/session.cpp

```diff
@@ -3698,9 +3701,7 @@ void Session::handleTorrentRemovedAlert(libt::torrent_removed_alert *p)
 
 void Session::handleTorrentDeletedAlert(libt::torrent_deleted_alert *p)
 {
-    const QString path = m_savePathsToRemove.take(p->info_hash);
-    if (path == torrentTempPath(p->info_hash))
-        Utils::Fs::smartRemoveEmptyFolderTree(path);
+    Utils::Fs::smartRemoveEmptyFolderTree(m_savePathsToRemove.take(p->info_hash));
```

Comment:
In fact, we have not needed to do this earlier. At this point, the folder only exists if it is a temporary subfolder.